### PR TITLE
Update to include missing strings to Croatian translation.

### PR DESCRIPTION
**Pull request by robimarko**
Added all missing strings according to English

### CODE REVIEW

File: Marlin/language_hr.h [hunk 1]
```diff
@@ -167,5 +175,154 @@
...
+#define MSG_INFO_THERMISTOR_MENU            "Termistori"
+#define MSG_INFO_EXTRUDERS                  "Extruderi"
+#define MSG_INFO_BAUDRATE                   "Baud"
+#define MSG_INFO_PROTOCOL                   "Protokol"
+#define MSG_LIGHTS_ON                       "Upali osvjetljenje"
+#define MSG_LIGHTS_OFF                      "Ugasi osvjetljenje"
+#if LCD_WIDTH >= 20
+  #ifndef MSG_INFO_PRINT_COUNT
+    #define MSG_INFO_PRINT_COUNT              "Broj printova"
+  #endif
```
Comment (thinkyhead) [inline Marlin/language_hr.h]: You will need to remove all these `#ifndef` lines. They are not supposed to be included in translations. Compare to other languages.

**Comment by robimarko** [comment]
Thanks,removed ifndef

**Comment by thinkyhead** [comment]
@robimarko We need shorter translations for the following strings. They must be 20  or fewer characters. And ideally 18 or fewer.
```cpp
                                           12345678901234567890|
#define MSG_AUTOSTART                     "Automatsko pokretanj|e"
#define MSG_INFO_PRINT_TIME               "Ukupno vrijeme print|anja"
#define MSG_INFO_PRINT_LONGEST            "Trajanje najdužeg pr|inta"
```
Any ideas?

**Comment by robimarko** [comment]
Yep.
Not really direct translation but understandable translations that are shorter are:
```cpp
#define MSG_AUTOSTART                     "Auto pokretanje"
#define MSG_INFO_PRINT_TIME               "Ukupno printanja"
#define MSG_INFO_PRINT_LONGEST            "Najduži print"
```
Do you want me to edit those and create pull request?

**Comment by thinkyhead** [comment]
Thanks! That'd be cool. Don't forget to fetch the latest `upstream` (i.e., `MarlinFirmware`) as there was another patch to the language file in the interim.

**Pull request closed without merging**